akka-cluster/src/main/resources/reference.conf

```diff
@@ -153,6 +153,17 @@ akka {
     # longer than this it could lead to removed nodes being re-added to the cluster
     prune-gossip-tombstones-after = 24h
 
+    multi-data-center {
+      # Try to limit the number of connections between data centers. Used for gossip and heartbeating.
+      # This will not limit connections created for the messaging of the application.
+      # If the cluster does not span multiple data centers, this value has no effect.
+      cross-data-center-connections = 5
```

Comment:
I think we should use the same setting or have it derived, because the whole idea is to gossip and failure detect the same nodes.